schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                            "TranslatedTitle"
+                        ]
+                    }
+                },
+                "required": [
+                    "title"
+                ]
+            }
+        },
+        "container": {
+            "description": "The container of the resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "The identifier for the container.",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "The type of the container.",
+                    "type": "string",
+                    "enum": [
+                        "Book",
+                        "BookSeries",
+                        "Journal",
+                        "Periodical",
+                        "ProceedingsSeries",
+                        "Series"
+                    ]
+                },
+                "title": {
+                    "description": "The title of the container.",
+                    "type": "string"
+                }
+            }
+        },
+        "contributors": {
+            "description": "The contributors to the resource.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "description": "The unique identifier for the contributor.",
+                        "type": "string"
+                    },
+                    "type": {
+                        "description": "The type of the contributor.",
+                        "type": "string",
+                        "enum": [
+                            "Organization",
+                            "Person"
+                        ]
+                    },
+                    "name": {
+                        "description": "The name of the contributor.",
+                        "type": "string"
+                    },
+                    "givenName": {
+                        "description": "The given name of the contributor.",
+                        "type": "string"
+                    },
+                    "familyName": {
+                        "description": "The family name of the contributor.",
+                        "type": "string"
+                    },
+                    "affiliation": {
+                        "$ref": "#/definitions/affiliations"
+                    }
+                }
+            }
+        },
+        "subjects": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "subject": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "subject"
+                ]
+            },
+            "uniqueItems": true
+        },
+        "sizes": {
```

Comment:
`sizes`, `formats`, `geo_location` and `subjects` could just be copied over into KBCMS as we don't currently have any similar fields

Reply:
`geo_location` is complex, and is implemented in DataCite metadata and schema.org. For subjects see above comment.